### DISCUSSION TITLE
Manual test AppImage with `appimage-tester` command - is this still work?

### DISCUSSION THREAD

**Discussion by spvkgn** [AppImage/AppImageKit#1324] [Q&A] [answered]
I'd like to test AppImage manually as described here https://appimage-builder.readthedocs.io/en/latest/advanced/testing.html
But did not find that file/command `appimage-tester` when install `appimage-builder` development version from git via `pip`. Or is this already obsolete?

**Comment by probonopd** [comment, marked as answer]
Hi @spvkgn, best ask over at https://github.com/AppImageCrafters/appimage-builder/issues because I don't think its developers are reading here. Thanks!
